Changelog.rst

```diff
@@ -9,7 +9,33 @@ in the main branch & 5.3.x series, please see :ref:`whatsnew-5.3` for
 an overview of what's new in Celery 5.3.
 
 
-. _version-5.3.5:
+.. _version-5.3.6:
+
+5.3.6
+=====
+
+:release-date: 2023-11-22  7:15 P.M GMT+6
+:release-by: Asif Saif Uddin
+
+What's Changed
+==============
+- Increased docker-build CI job timeout from 30m -> 60m by @Nusnus in https://github.com/celery/celery/pull/8635
+- Incredibly minor spelling fix. by @Asday in https://github.com/celery/celery/pull/8649
+- Fix non-zero exit code when receiving remote shutdown by @lyzlisa in https://github.com/celery/celery/pull/8650
+- Update task.py get_custom_headers missing 'compression' key by @auvipy in https://github.com/celery/celery/pull/8633
+- Update kombu>=5.3.4 to fix SQS request compatibility by @auvipy in https://github.com/celery/celery/pull/8646
```

Comment:
Hi @Nusnus, were you referring to this fix for the patch?

Reply:
Not only that—I'm not fully on the details, but across several issues, there was a discussion explaining what was wrong, why, and the solution. A brief summary with ref to PRs could help communicate the outline of the changes, to save the community the investigation afterwards if anyway's interested to understand it.

P.S
IIRC there was a format change to `json`, so a small note about it could do a lot IMHO

Reply:
Thanking you for throwing in a bit of spotlight there, got it. Yes, I second that motive. Concise summaries with changes refs. is the best way forward. 👍🏼